common/lib/capa/capa/capa_problem.py

```diff
@@ -950,21 +950,22 @@ def response_a11y_data(self, response, inputfields, responsetype_id, problem_dat
 
                 label = inputfields[0].attrib['label']
                 # Get first <p> tag before responsetype, this <p> contains the question text.
-                p_tag = response.xpath('preceding-sibling::p[1]')
+                p_tag = response.xpath('preceding-sibling::p[1]')[0]
```

Comment:
@ehteshamkafeel this will crash if `response.xpath('preceding-sibling::p[1]')` returns an empty list.